Base/Models/Public/Include/Models/UnifyedModel.hpp

```diff
@@ -32,8 +32,8 @@ class UnifyedModel
     size_t                   _amountOfFields;
 
 protected:
-    Map<TEnum>       _data;
-    Map<TEnum>       _fieldData;
+    mutable Map<TEnum>       _data;
```

Comment:
It's for the future. In our methods we transfer data as const reference. In this it comes to the method where database logic exists. And in some cases we need to add more information about the model, but we can't bc of const reference. It means that we need to
1. Forget about it and leave [0][0].as<type>() casts everywhere (that's not a good solution)
2. Create in this method another one object, which will copy incoming information from the model and additional, and it's going to be const or smth like it. Just a new model, more memory (I see it as too expensive solution)
3. Add this additional information to the const reference without any new models or staying kinda weird casts.
Maybe there is another way how to solve it much better
![image](https://user-images.githubusercontent.com/63643593/235125643-838e11cc-e5da-4546-8d0c-427a4dcacc0c.png)
![image](https://user-images.githubusercontent.com/63643593/235125722-5d98cd3f-e83f-479a-8297-9564e18912b0.png)
P.S Ofc that's the same method and as we see. Data is separated (id) from the model and we can't add this to it even because there's no such field in the model. And I'm sure that I can find more examples of this
P.S.S This file is terrible